app.js

```diff
@@ -3,45 +3,76 @@ const jwt = require('jsonwebtoken')
 const app = express()
 const port = 8080
```

Comment:
For consistency with the rest of the class, it would be a good idea to change this to port 3000.